packages/@lwc/engine-dom/src/apis/create-element.ts

```diff
@@ -81,17 +81,25 @@ function monkeyPatchDomAPIs() {
     } as Pick<Node, 'appendChild' | 'insertBefore' | 'removeChild' | 'replaceChild'>);
 }
 
+// For some reason, JSDOC says "options.is" is a syntax error. And we can't disable the rule using
+// `eslint-disable-next-line` because that gets included in the JSDOC, so we need this workaround.
+/* eslint-disable jsdoc/valid-types */
 /**
  * EXPERIMENTAL: This function is almost identical to document.createElement with the slightly
```

Comment:
I don't know why we call this EXPERIMENTAL. It's used all over the place, especially in Jest tests.